mne_bids_pipeline/steps/preprocessing/_04_frequency_filter.py

```diff
@@ -64,6 +65,29 @@ def get_input_fnames_frequency_filter(
     )
 
 
+def zapline(
+    raw: mne.io.BaseRaw,
+    subject: str,
+    session: str | None,
+    run: str,
+    task: str | None,
+    fline: float,
+    iter_: bool,
+) -> None:
+    """Use Zapline to remove line frequencies."""
+    if fline is None:
+        return
```

Comment:
@drammock there is a short-circuit here for the `None` case

Reply:
> there is a short-circuit here for the None case

🤦🏻 sorry, how did I miss that.